examples/todo-mvc/ui/views/task_list_view.slint

```diff
@@ -61,6 +62,7 @@ export component TaskListView {
 
             ActionButton {
                 icon: Icons.add;
+                accessible-label: @tr("add");
```

Comment:
A bit more context could be useful. "Create new task" or something along those lines would be clearer.